tests/findComponent.spec.ts

```diff
@@ -37,7 +42,46 @@ const compA = defineComponent({
   components: { compB, Hello }
 })
 
+const createRenderComponent = (
+  name: string,
+  props: Record<string, string> = {},
+  children?: any
+) => ({
+  name,
+  render() {
+    return h('div', props, children)
+  }
+})
+
 describe('findComponent', () => {
+  it('works with simple render functions', () => {
+    const Child = {
+      name: 'Child',
+      render() {
+        return h('div', { id: 'child' })
+      }
+    }
+    const Parent = {
+      name: 'Parent',
+      render() {
+        return h(Child)
+      }
+    }
+
+    const wrapper = mount(Parent)
+    wrapper.getComponent(Child)
```

Comment:
I am confused by `getComponent` used in this context (and some following lines as well).

Reply:
I think I will abandon this PR for now since @Jokcy found a better solution.

I was just doing this as a quick test - the idea is that if the component is *not* found, it will throw an error (`findComponent` would *not*).

Reply:
I see. Just for my understanding, #188 is solving the slots case but not the functional components one?

If this is true, should we pursue your proposal of having some warning or anything else?

Reply:
I would say so. This would be a much better developer experience.

We could use this `__isFunctionalComponent` property I am proposing we add here https://github.com/vuejs/vue-test-utils-next/pull/185

What do you think?

Reply:
I like it!